example/.mage/tools.mk

```diff
@@ -1,49 +1,11 @@
 mage_folder := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
-mage_generated_path := $(mage_folder)/tools/mgmake
 mage_tools_path := $(mage_folder)/tools
-mage_targets_file := $(mage_generated_path)/targets.mk
-mage := $(mage_generated_path)/local-mage
-mgmake := $(mage_folder)/mgmake_gen.go
-mgmain := $(mage_folder)/mgmain_gen.go
+mage := $(mage_tools_path)/mgmake/magefile
+version := $(shell cd $(mage_folder) && go list -m -f '{{ .Version }}' go.einride.tech/mage-tools)
```

Comment:
let's call this `mage_version` to avoid naming collissions when this file is `include`:d

Reply:
As we mentioned below, version is not needed